CMS-Enterprise/easi-app
# [NOREF] Upgrade axios

# NOREF

## Description

- Upgrades `axios` package

## How to test this change

We only use axios in a handful of places (shown below). I'd recommend potentially just testing any functionality related to these sections.
![image](https://github.com/user-attachments/assets/42e986fc-552c-4a2f-b4d8-4bd92c11aace)

The most critical sections to test are likely:
- Downloading Files (Documents)
- The business case flow (create, edit, submit -- should be tested by E2E tests)

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
